src/main/webapp/index.html

```diff
@@ -1,68 +1,130 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-      name="viewport"
-    />
-    <meta content="STEP Capstone: Techish" name="description" />
-    <meta content="Samuel Sandoval and Hadley Shapland" name="author" />
-    <title>Techish</title>
-    <link href="css/styles.css" rel="stylesheet" />
-  </head>
-  <body onload="loadMatches();">
-    <!-- navigation bar -->
-    <div class="nav-item" id="login-status-section" style="display: block;">
-      <p>Link:</p>
-      <a class="login-text" id="login-link" href="#"></a>
+<html lang="en">
+
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="Techish Web Application">
+  <meta name="author" content="Samuel Sandoval and Hadley Shapland">
+
+  <title>Techish: Mentor Matching</title>
+
+  <!-- Bootstrap core CSS -->
+  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Custom fonts for this template -->
+  <link href="https://fonts.googleapis.com/css?family=Catamaran:100,200,300,400,500,600,700,800,900" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i" rel="stylesheet">
+
+  <!-- Custom styles for this template -->
+  <link href="css/one-page-wonder.min.css" rel="stylesheet">
+
+</head>
+
+<body>
+
+  <!-- Navigation -->
+  <nav class="navbar navbar-expand-lg navbar-dark navbar-custom fixed-top">
+    <div class="container">
+      <a class="navbar-brand" href="#">Techish</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarResponsive">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="profileBuilder.html">Sign Up</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="login-link" href="logged_in_homepage.html">Log In</a>
```

Comment:
nit: revert the log in to the # so the UserAPI feature works.

Then, make another navbar link that directs to the logged in home page html page

Reply:
I fixed the link. I'm not sure if the login part is working correctly, but once I merge this you can change it so that it works with your UserAPI.